internal/pkg/cli/validate.go

```diff
@@ -13,12 +13,15 @@ import (
 )
 
 var (
-	errValueEmpty      = errors.New("value must not be empty")
-	errValueTooLong    = errors.New("value must not exceed 255 characters")
-	errValueBadFormat  = errors.New("value must start with a letter and contain only lower-case letters, numbers, and hyphens")
-	errValueNotAString = errors.New("value must be a string")
+	errValueEmpty        = errors.New("value must not be empty")
+	errValueTooLong      = errors.New("value must not exceed 255 characters")
+	errValueBadFormat    = errors.New("value must start with a letter and contain only lower-case letters, numbers, and hyphens")
+	errValueNotAString   = errors.New("value must be a string")
+	errInvalidGitHubRepo = errors.New("value must be a valid GitHub repository, e.g. https://github.com/myCompany/myRepo")
 )
 
+var githubRepoExp = regexp.MustCompile(`https:\/\/github\.com\/(?P<owner>.+)\/(?P<repo>.+)`)
```

Comment:
Does CodePipeline support ssh style endpoints for GitHub repositories? (e.g. `git@github.com:aws/amazon-ecs-cli-v2.git`)